ftl/BUILD

```diff
@@ -43,3 +47,35 @@ py_test(
         "@mock//:mock",
     ],
 )
+
+py_library(
+    name = "node_lib",
+    srcs = glob(["node/*.py"]),
+    deps = [":ftl_lib"],
+)
+
+# The node base image sets a default CMD, which causes issues if you run our image with no CMD.
+# So, reset it to [""] before building our py_image.
+docker_build(
+    name = "node_builder_base",
+    base = "@node_base//image",
+    cmd = [""],
+    entrypoint = "",
+)
+
+py_binary(
+    name = "node_builder",
+    srcs = [":node_lib"],
+    main = "node/main.py",
+    deps = ["@containerregistry"],
+)
```

Comment:
Pro tip: you can just `bazel run :node_builder_image.binary`

Reply:
Nice!